lib/igniter.ex

```diff
@@ -586,7 +587,7 @@ defmodule Igniter do
             {true,
              ""
              |> Rewrite.Source.Ex.from_string(path)
-             |> Rewrite.Source.update(:file_creator, :content, contents)}
+             |> update_source(igniter, :content, contents, by: :file_creator)}
```

Comment:
I believe the previous call format was already incorrect and should have been `by: :file_creator` -- something that slipped by in the update to Rewrite 1.0?

The next change is the same.